libs/dh/admin/data-access-api/src/lib/dh-admin-user-management-data-access-api.store.ts

```diff
@@ -57,6 +57,11 @@ export class DhAdminUserManagementDataAccessApiStore extends ComponentStore<DhUs
   users$ = this.select((state) => state.users);
   totalUserCount$ = this.select((state) => state.totalUserCount);
 
+  // 1 needs to be substracted here because our endpoint's `pageNumber` param starts at `1`
+  // whereas the paginator's `pageIndex` property starts at `0`
+  paginatorPageIndex$ = this.select((state) => state.pageNumber - 1);
```

Comment:
question: seems to me like we should either make the endpoint 0-based or perhaps add a `pageNumber` input to the new `watt-paginator` to make the component 1-based instead. what do you think?

Reply:
I've added a `pageIndex` input to the `DhSharedUiPaginatorComponent` component (as seen in the last file in this PR). We should definitely add something similar to `watt-paginator`. But since `watt-paginator` is still based on Mat Paginator, I doubt we can simply add a `pageNumber`/`pageIndex` property that is 1-based. The conversion from 1 to 0 should still happen somewhere. It can be internal to `watt-paginator` though.

Without thinking about it too deeply, I'd say 1-based pagination feels more natural. Mat Paginator is 0-based probably because of something technical. Hopefully no one says "Give me page 0" 😄